contract-ui/tabs/claim-conditions/components/claim-conditions-form/index.tsx

```diff
@@ -252,6 +376,38 @@ export const ClaimConditionsForm: React.FC<ClaimConditionsFormProps> = ({
     }
   });
 
+  const activePhaseId = useMemo(() => {
+    let phaseId: string | null = null;
+    let latestStartTime: number = 0;
+
+    controlledFields.forEach((phase) => {
+      if (!phase.startTime) return;
+
+      const phaseStartTime =
+        typeof phase.startTime === "object"
+          ? phase.startTime.getTime()
+          : phase.startTime;
+      const currentTime = new Date().getTime();
+
+      if (phaseStartTime < currentTime && phaseStartTime > latestStartTime) {
+        latestStartTime = phaseStartTime;
+        phaseId = phase.id;
```

Comment:
we can break the loop and return here. ( need to convert forEach => for loop for that )